src/main/webapp/app/exercises/shared/exercise-detail-common-actions/non-programming-exercise-detail-common-actions.component.html

```diff
@@ -71,10 +77,5 @@
         <span style="margin-right: 5px">
             <jhi-exercise-submission-export *ngIf="exercise.isAtLeastInstructor" [exerciseId]="exercise.id!" [exerciseType]="exercise.type!"> </jhi-exercise-submission-export>
         </span>
```

Comment:
The extra margin on the left of this button happens because jhi-external-submission is not displayed but span of it is still there with margin-right:5px. It is line 72-74 with the changes.

Reply:
Will keep this in mind when handling this in the future